utilities/hco.py

```
@@ -84,9 +84,14 @@ def __init__(
         self.wait_for_reconcile_post_update = wait_for_reconcile_post_update
         self._consecutive_checks_count = consecutive_checks_count
         self.list_resource_reconcile = list_resource_reconcile or []
+        self._baseline_tainted: bool = False
         LOGGER.info(f"Patches: {self.patches}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)